File: yht0827/2월/2월 13일/JadenCase 문자열 만들기.java
class Solution {
	public String solution(String s) {
		String answer = "";
		String[] sArr = s.toLowerCase().split("");
		StringBuilder sb = new StringBuilder();

		for (int i = 0; i < sArr.length; i++) {
			if (i == 0)
				sArr[i] = sArr[i].toUpperCase();

			if (" ".equals(sArr[i])) {
				if (!(i == sArr.length - 1))
					sArr[i + 1] = sArr[i + 1].toUpperCase();
			}

			sb.append(sArr[i]);
		}

		answer = sb.toString();

		return answer;
	}
}

File: Eunleelee/2024-02/2월 13일/JadenCase 문자열 만들기.py
def solution(s):
    s_list = s.split(' ')
    answer = ''
    num = 0
    for i in range(len(s_list)):
        s_list[i] = s_list[i].capitalize()
        answer += s_list[i]
        if i != len(s_list) - 1:
            answer += ' '
            
    return answer
